systems/summit_user_guide.rst

```diff
@@ -1662,8 +1662,196 @@ contain 6 MPI tasks (``-a6``), 3 GPUs (``-g3``), and 6 cores
+    * ``bsub -W 10 -nnodes 2 -P $OLCF_PROJECT_ID -Is $SHELL``
+2. Load the ``job-step-viewer`` module
+    * ``module load job-step-viewer``
+3. Test out a ``jsrun`` line by itself, or provide an executable as normal
+    * ``jsrun -n12 -r6 -c7 -g1 -a1 EOMP_NUM_THREADS=7 -brs``
+4. Visit the provided URL
+    * https://jobstepviewer.olcf.ornl.gov/summit/871957-1
+
+.. note::
+    Most Terminal applications have built-in shortcuts to directly open
+    web addresses in the default browser.
+
+    * MacOS Terminal.app: hold Command (⌘) and double-click on the URL
+    * iTerm2: hold Command (⌘) and single-click on the URL
+
+Limitations
+___________
+
+* (currently) Compiled with GCC toolchain only
+* Does not support MPMD-mode via ERF
+* OpenMP only supported with use of the ``OMP_NUM_THREADS`` environment variable.
+
+
+More Information
+^^^^^^^^^^^^^^^^
+
+This section provides some of the most commonly used LSF commands as
+well as some of the most useful options to those commands and
+information on ``jsrun``, Summit's job launch command. Many commands
+have much more information than can be easily presented here. More
+information about these commands is available via the online manual
+(i.e. ``man jsrun``). Additional LSF information can be found on `IBM’s
+website <https://www.ibm.com/support/knowledgecenter/en/SSWRJV/product_welcome_spectrum_lsf.html>`__.
+
+
+Using Multithreading in a Job
+-----------------------------
+
+
+Hardware Threads: Multiple Threads per Core
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each physical core on Summit contains 4 hardware threads. The SMT level
+can be set using LSF flags (the default is smt4):
+
+SMT1
+
+::
+
+    #BSUB -alloc_flags smt1
+    jsrun -n1 -c1 -a1 -bpacked:1 csh -c 'echo $OMP_PLACES’
+    0
+
+SMT2
+
+::
+
+    #BSUB -alloc_flags smt2
+    jsrun -n1 -c1 -a1 -bpacked:1 csh -c 'echo $OMP_PLACES’
+    {0:2}
+
+SMT4
+
+::
+
+    #BSUB -alloc_flags smt4
+    jsrun -n1 -c1 -a1 -bpacked:1 csh -c 'echo $OMP_PLACES’
+    {0:4}
+
+
+
+
+
+Controlling Number of Threads for Tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to specifying the SMT level, you can also control the
+number of threads per MPI task by exporting the ``OMP_NUM_THREADS``
+environment variable. If you don't export it yourself, Jsrun will
+automatically set the number of threads based on the number of cores
+requested (``-c``) and the binding (``-b``) option. It is better to be
+explicit and set the ``OMP_NUM_THREADS`` value yourself rather than
+relying on Jsrun constructing it for you. Especially when you are
+using `Job Step Viewer`_ which relies on the presence of that
+environment variable to give you visual thread assignment information.
+
+In the below example, you could also do ``export OMP_NUM_THREADS=16`` in your
+job script instead of passing it as a ``-E`` flag to jsrun. The below example
+starts 1 resource set with 2 tasks and 8 cores, 4 cores bound to each task,
+16 threads for each task.
+
```

Comment:
Since you say below this why you can't do 32 threads, I would include somewhere around here why you're able to do 16 threads -- a.k.a 4 * 4  = 16 because SMT4 and 4 cores